Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/LocalData.kt

```diff
@@ -399,6 +401,12 @@ object LocalData {
         return Date(time)
     }
 
+    fun lastTimeDiagnosisKeysFromServerFetchFlow() =
+        getSharedPreferenceInstance()
+            .createFlowPreference<Long?>(CoronaWarnApplication.getAppContext()
+                .getString(R.string.preference_timestamp_diagnosis_keys_fetch), 0L).flow
```

Comment:
why are we using a key from string resources here, not const val?

Reply:
That seems to be the way we deal with all keys in LocalData. Just using what we already have in place  - see e..g the getter above.

Reply:
Got this questions a few weeks ago also from @d4rken , in the beginning we followed the example from the docu: https://developer.android.com/training/data-storage/shared-preferences#GetSharedPreferences

![image](https://user-images.githubusercontent.com/64483219/99536225-5d370c80-29aa-11eb-8fd6-075133fe3f64.png)

But I would say that's something for a future refactoring.